clue.js

```diff
@@ -0,0 +1,340 @@
+  image: 'assets/poison.png'
+}
+
+const horseshoe = {
+  name: 'Horseshoe',
+  color: '#bab2ab',
+  description: 'What suspect brings his own horses to the country?',
+  image: 'assets/horseshoe.png'
+}
+
+const hatPin = {
+  name: 'Hat Pin',
+  color: '#a2c3a3',
+  description: 'Small, sharp, and discreet.',
+  image: 'assets/hatpin.png'
+}
+
+const wrench = {
+  name: 'Wrench',
+  color: '#d1d194',
+  description: 'Whose fingerprints are on the grip?',
+  image: 'assets/wrench.png'
+}
+
+const revolver = {
+  name: 'Revolver',
+  color: '#ebccad',
+  description: 'The mechanism seems well-oiled...',
+  image: 'assets/revolver.png'
+}
+
+// Objects for Rooms
+const hall = {
+  name: 'Hall',
+  color: '#c29970',
+  image: 'assets/hall.png'
+}
+
+const lounge = {
+  name: 'Lounge',
+  color: '#d19494',
+  image: 'assets/lounge.png'
+}
+
+const diningRoom = {
+  name: 'Dining Room',
+  color: '#669966',
+  image: 'assets/diningroom.png'
+}
+
+const kitchen = {
+  name: 'Kitchen',
+  color: '#dbbdcc',
+  image: 'assets/kitchen.png'
+}
+
+const ballroom = {
+  name: 'Ballroom',
+  color: '#bdbddb',
+  image: 'assets/ballroom.png'
+}
+
+const conservatory = {
+  name: 'Conservatory',
+  color: '#92b2c9',
+  image: 'assets/conservatory.png'
+}
+
+const billiardRoom = {
+  name: 'Billiard Room',
+  color: '#667a52',
+  image: 'assets/billiardroom.png'
+}
+
+const library = {
+  name: 'Library',
+  color: '#738c80',
+  image: 'assets/library.png'
+}
+
+const study = {
+  name: 'Study',
+  color: '#7ab8b8',
+  image: 'assets/study.png'
+}
+
+const spa = {
+  name: 'Spa',
+  color: '#ccd1c7',
+  image: 'assets/spa.png'
+}
+
+const observatory = {
+  name: 'Observatory',
+  color: '#667f99',
+  image: 'assets/observatory.png'
+}
+
+const stables = {
+  name: 'Stables',
+  color: '#a6a659',
+  image: 'assets/stables.png'
+}
+
+const guestHouse = {
+  name: 'Guest House',
+  color: '#d6c2d6',
+  image: 'assets/guesthouse.png'
+}
+
+const trophyRoom = {
+  name: 'Trophy Room',
+  color: '#999966',
+  image: 'assets/trophyroom.png'
+}
+
+const carriageHouse = {
+  name: 'Carriage House',
+  color: '#b8cce0',
+  image: 'assets/carriagehouse.png '
+}
+
+// Arrays for suspects, weapons, and rooms.
+const suspects = [
+  mrGreen, 
+  profPlum, 
+  missScarlett, 
+  mrsPeacock, 
+  colMustard, 
+  mrsWhite
+]
+
+const weapons = [
+  rope, 
+  knife, 
+  candlestick, 
+  leadPipe, 
+  poison, 
+  horseshoe, 
+  hatPin, 
+  wrench, 
+  revolver
+]
+
+const rooms = [
+  hall, 
+  lounge, 
+  diningRoom, 
+  kitchen, 
+  ballroom, 
+  conservatory, 
+  billiardRoom, 
+  library, 
+  study, 
+  spa, 
+  observatory, 
+  stables, 
+  guestHouse, 
+  trophyRoom, 
+  carriageHouse
+]
+
+// THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
+const randomSelector = array => {
+  return array[Math.floor(Math.random() * array.length)];
+}
+
+// Object to hold The Mystery
+const mystery = {
+  killer: '',
+  weapon: '',
+  room: ''
+}
```

Comment:
Let's prefer `null`s here.  The key to this thought is:
What values will these variables have later?  If they will become objects, it can be troublesome if they start as strings.  

`null` is standard for most variables that don't yet have any value

Reply:
updated!